bc_obps/reporting/models/facility_report.py

```diff
@@ -1,16 +1,31 @@
 from django.db import models
 from registration.models import ReportingActivity, RegulatedProduct
+from registration.models.facility import Facility
 from registration.models.time_stamped_model import TimeStampedModel
 from reporting.models import ReportVersion
 
 
-class ReportFacility(TimeStampedModel):
+class FacilityReport(TimeStampedModel):
+    '''
+    Model representing a report for a single facility.
+    A report (each report version) may contain multiple facility reports.
+    '''
+
+    facility = models.ForeignKey(
+        Facility,
+        on_delete=models.DO_NOTHING,
+        null=True,
+        blank=True,
+        db_comment="""The facility record this report was created for, at the time the report was filled out.
+        This may be null in the future if the facility disappears or changes ownership.""",
```

Comment:
Will it be null if the facility changes ownership? I think the relationship will remain in that case since the ID of the facility will remain the same.

Reply:
That makes sense! I didn't really have a good idea of the lifecycle of those things in the registration app. I'll remove the Null/Blank options